tests/data/placebo/test_ec2_instance_age_minute_filter/ec2.DescribeInstances_1.json

```diff
@@ -0,0 +1,149 @@
+{
+    "status_code": 200, 
```

Comment:
there aren't any test cases that correspond to these recorded test files. can they just be removed?